cirq-google/cirq_google/devices/serializable_device.py

```diff
@@ -108,6 +108,25 @@ def __init__(
         """
         self.qubits = qubits
         self.gate_definitions = gate_definitions
+        self._metadata = cirq.GridDeviceMetadata(
+            [
+                (pair[0], pair[1])
```

Comment:
I would probably make this keyword rather than positional arguments for clarity.

Reply:
Done.